src/services/cli-http-client.js

```diff
@@ -64,12 +65,16 @@ class CliRequestClient {
       const b64Auth = Buffer.from(`${opts.username}:${opts.password}`).toString('base64');
       headers.Authorization = `Basic ${b64Auth}`;
     }
-
-    const componentInfo = (headers['User-Agent'] || '').replace(' (', '|').replace(')', '').split('|');
-    componentInfo.push(`${os.platform()} ${os.release()} ${os.arch()}`);
+    const componentInfo = [];
+    componentInfo.push(`(${os.platform()} ${os.arch()})`);
+    const userAgentArr = (headers['User-Agent'] || '').split(' ');
```

Comment:
[Q] Is there any reason to split with '' here and with ' ' [there](https://github.com/twilio/twilio-cli-core/pull/140/files#diff-45f5550d8999c92e19e96dbf464fe1c9012ea83c1368b7d6fcfa1122bdaa0c55R29)?

Reply:
It's always split with ' ' in both the places. With the || `(headers['User-Agent'] || '')`, I'll make it ' ' (with space) there.

Reply:
It's done.